toolkit/tools/pkg/imagecustomizerlib/bootcustomizer.go

```diff
@@ -211,3 +211,11 @@ func (b *BootCustomizer) WriteToFile(imageChroot safechroot.ChrootInterface) err
 
 	return nil
 }
+
+func (b *BootCustomizer) GetDefaultGrubFileContent() string {
```

Comment:
The `GetDefaultGrubFileContent` and `SetDefaultGrubFileContent` functions are breaking the abstraction of `BootCustomizer`. Add a `SetRootDevice` function instead.